src/lib.rs

```diff
@@ -383,8 +384,8 @@ fn pick_mesh(
             let vertex_positions: Vec<[f32; 3]> = mesh
                 .attributes
                 .iter()
-                .filter(|attribute| attribute.name == VertexAttribute::POSITION)
```

Comment:
attribute.name has been replaced with the first item in a tuple of `(str, VertexAttributeValues)`